html-css/portfolio.css

```diff
@@ -0,0 +1,136 @@
+
+body {
+    font-family: Arial, sans-serif;
+    color: #333;
+    background-color: #fafafa;
+    line-height: 1.6;
+    display: flex;
```

Comment:
Remove "display: flex" to allow content to flow naturally unless you need a centered container but it could be your preference.